og.og_permissions.yml

```diff
@@ -2,10 +2,11 @@ update group:
   title: 'Edit group'
   description: 'Edit the group. Note: This permission controls only node entity type groups.'
   'default role':
-    - OG_ADMINISTRATOR_ROLE
```

Comment:
> Any pointer to those 'special' strings?

Oh, we could just process them ourself in `\Drupal\og\OgPermissionHandler::buildPermissionsYaml` - is that what you meant? Anyway, we'll still have re-define the constants in the test file, for the unit test to pass